crates/languages/src/go.rs

```diff
@@ -43,6 +43,12 @@ static GO_ESCAPE_SUBTEST_NAME_REGEX: LazyLock<Regex> = LazyLock::new(|| {
     Regex::new(r#"[.*+?^${}()|\[\]\\]"#).expect("Failed to create GO_ESCAPE_SUBTEST_NAME_REGEX")
 });
 
+const BINARY_DIR: &str = if cfg!(target_os = "windows") {
```

Comment:
This should just be `BINARY`, as it's not an actual directory.

```suggestion
const BINARY: &str = if cfg!(target_os = "windows") {
```

Reply:
I could rename it to BINARY_GOPLS if that's better.

Reply:
> I could rename it to BINARY_GOPLS if that's better.

I think just `BINARY` is fine, thanks!